### DISCUSSION TITLE
git pull error: Your local changes to the following files would be overwritten by merge

### DISCUSSION THREAD

**Discussion by U--d--o** [Koenkk/zigbee2mqtt#12642] [General]
all went well, however on today's git pull he doesn't like "local changes". where i am puzzled since these changes can only have occurred by using the program. Changes through use should be taken into account in the project, or do I understand something wrong?

C:\zigbee2mqtt>git pull
error: Your local changes to the following files would be overwritten by merge:
.github/workflows/ci.yml
.github/workflows/update_deps.yml
.github/workflows/update_frontend.yml
.github/workflows/update_zh.yml
.github/workflows/update_zhc.yml
docker/Dockerfile
index.js
lib/extension/homeassistant.ts
lib/extension/networkMap.ts
lib/extension/otaUpdate.ts
lib/util/settings.schema.json
npm-shrinkwrap.json
package.json
Please commit your changes or stash them before you merge.
Aborting

**Comment by Koenkk** [comment]
What is the output of `git diff` ?

**Reply by U--d--o** [reply]
unbelievable 36254 lines of output.
 
The name changes of actors are also displayed. Unfortunately, i do not see any meaningful restriction that distinguishes the application from the user data.

**Reply by Koenkk** [reply]
Maybe windows changed the line endings or something like that. I would recommend to make a new clone and copy your `data` directory into that.
